stable/influxdb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: influxdb
-version: 1.5.0
+version: 1.5.1
```

Comment:
As we discussed, plase bump the major version.

Reply:
Done!